src/Uno.UWP/UI/Popups/MessageDialog.macOS.cs

```diff
@@ -0,0 +1,68 @@
+﻿#if __MACOS__
```

Comment:
Not sure it's required when the file is suffixed `.macos.cs`.  @jeromelaban ?

Reply:
It's not required indeed, the cue is when the project selector does not show it if macOS is not active for the current build because of the override.